gcloud/datastore/transaction.py

```diff
@@ -89,34 +89,13 @@ class Transaction(Batch):
       >>> transaction.begin()
 
       >>> entity = Entity(key=Key('Thing'))
-      >>> entity.save()
+      >>> datastore.put([entity])
```

Comment:
This won't actually be stored in the transaction since `Batch.__enter__` pushes to the stack, not `Transaction.begin()`.